client/src/components/roulette/RouletteWheel.jsx

```diff
@@ -110,44 +137,37 @@ export default function RouletteWheel({ betInfo }) {
           textDistance={textDistance}
           onStopSpinning={() => {
             setMustSpin(false);
-            axios.get('/api/roulette', {
-              params: {
-                betInfo: betInfo,
-                winNum: prizeNumber
-              }
-            })
-              .then(results => {
-                // need to clean out the current bets
-                if (results.data) {
-                  (setWinState(true))
-                  // window.alert(`Congratulations! You won a total of ${results.data} dollars!`)
-                  //update global userbalance hook here
-                } else {
-                  window.alert('You did not win this time 👉👈 ')
-                }
-              })
-              .catch(err => console.log(err));
+            setSpin(!spin);
+            const { updatedBalance, winAmount } = winData;
+            if (winAmount) {
+              window.alert(`Congratulations! You won a total of ${winAmount} dollars!`);
```

Comment:
there is a main branch update here to show animation instead of alert. Maybe pull down and commit again?